lib/seattleflu/db/cli/command/manifest.py

```diff
@@ -51,7 +51,7 @@ def manifest():
 
 @click.option("--sample-column",
     metavar = "<column>",
-    help = "Name of the single column containing sample barcodes.  "
+    help = "Name of the single column containing sample or kit barcodes.  "
```

Comment:
The thought here was that this `--sample-column` is the only required column option and the only required column for the kit manifest would be the kit barcodes.

Reply:
Nod. Thoughts on alternatively adding `--kit-column` and declaring it and `--sample-column` both be optional, but requiring at least one is present at command runtime?

This would also translate to the ETL process looking for a `kit` vs. `sample` key.